doc/rtcp-termination.md

```diff
@@ -1,221 +1,94 @@
-	      'termination.strategies.HighestQualityRTCPTerminationStrategy')
-
-using the JavaScript Console, for example. The other 4 strategies
-mentioned in the previous update can be enabled/disabled similarly :
-
-    focus.setRTCPTerminationStrategy('org.jitsi.impl.neomedia.rtcp.' +
-	      'termination.strategies.STRATEGY_NAME')
+## Introduction
+
+
+The Jitsi Videobridge (JVB) is a Selective Forwarding Unit (SFU) (or a Selective
+Forwarding Middlebox as per [[RFC7667]] nomenclature) and as such it
+_terminates_ RTCP [[RFC3550]] traffic; the endpoints of a group call are
+"tricked" to believe that that are in a one-on-one call with the JVB. This has
+the notable desirable property that _bad_ down-link conditions at a particular
+receiver do not typically affect the sending bitrate of the senders because,
+from their perspective, there is only one receiver (the JVB) with a presumably
+good down-link.
+
+The JVB, in its turn, estimates its available up-link bandwidth [[BWE]] towards
+a particular receiver (or, symmetrically, the available down-link of a
+particular receiver) and it distributes it [[BWD]] among the several video
+tracks that the several senders of a group-call are sharing. This results in a
+bitrate allocation for each track of each sender for each receiver in a call.
+Each video track is _projected_ [[RFC7667], Section-3.7] differently onto each
+receiver, depending on its bitrate allocation and its adaptivity features such
+as simulcast [[SIMULCAST]], scalable video coding [[SVC]] and last-N [[LASTN]].
+
+While RTCP termination and bandwidth distribution are essential group
+conferencing features they are not desirable and they may degrade the quality of
+service in one-on-one calls. Unfortunately, there is no way to them at the
+moment, i.e. it is not possible to make the JVB behave as a translator
+[[RFC7667]], not even for one-on-one calls. Due to this limitation it is
+recommended to use a TURN server [[TURN]] to relay one-to-one calls for optimal
+conferencing experience.
+
+In the next few paragraphs, we'll describe how RTCP termination works exactly
+and which components are responsible for which task. For an overview of the
+media transport aspects of the WebRTC framework and RTP usage
+[[I-D.ietf-rtcweb-rtp-usage]] is highly recommended.
+
+## RTCP for bandwidth estimations
+
+RTCP Sender Reports (SRs) [[RFC3550], Section 6.4.1] are used by media senders
+to compute RTT and by media receivers for A/V syncing and for computing packet
+loss. In its media sender role the JVB projects SSRCs for bandwidth adaptivity,
+it needs to produce its own SRs. RTCP SR generation takes place in the various
+`AdaptiveTrackProjectionContext` implementations (see, for example,
+`BasicAdaptiveTrackProjectionContext#rewriteRtcp`).
+
+For estimating the available up-link bandwidth towards a particular receiver
+(or, symmetrically, the available down-link bandwidth of a particular receiver)
```

Comment:
i'm glad you also phrase these as from the receiver's perspective.  to me, the link which involves packets flowing from the bridge to the receiver is best described as 'the receiver's downlink'--i think calling it the bridge's uplink, though correct, is confusing.  my vote would be to exclusively use the 'receiver's downlink' terminology and not use the other one at all, but just my opinion...i'm thankful you use both at least!

Reply:
I have made the receivers downlink terminology the primary terminology and kept the bridge uplink terminology in the parenthesis. We can remove it if it feels meh